pkg/networkmanager/firewall.jsx

```diff
@@ -515,6 +524,9 @@ class AddServicesModal extends React.Component {
                                 </fieldset>
                             }
                         </form>
+                        {
+                            this.state.dialogError && <ModalError dialogError={this.state.dialogError} dialogErrorDetail={this.state.dialogErrorDetail} />
```

Comment:
I tend to put the errors in the footer, so that it will create this visual spacing between the error sections and the dialog inputs.
But it's just a personal preference, I am totally fine with this.
Just in case you wondered why my screenshots has more spacing between the error and the input fields.

Reply:
Let's make it consistent with the other Cockpit dialogs indeed. Thanks for spotting!

Reply:
I'll do that!